src/uu/tail/src/tail.rs

```diff
@@ -458,6 +468,9 @@ fn uu_tail(mut settings: Settings) -> UResult<()> {
                         unbounded_tail(&mut reader, &settings)?;
                     }
                     if settings.follow.is_some() {
+                        if let Some((ref mut watcher, _)) = watcher_rx {
```

Comment:
Can we move this one before tail is actually printed, so that we don't have anytime that event can happen in between?